tedana/combine.py

```diff
@@ -158,23 +158,32 @@ def make_optcom(data, tes, mask, t2s=None, combmode='t2s', verbose=True):
         LGR.warning("Argument 't2s' is not required if 'combmode' is 'paid'. "
                     "'t2s' array will not be used.")
 
-    data = data[mask, :, :]  # mask out empty voxels/samples
-    tes = np.array(tes)[np.newaxis, ...]  # (1 x E) array_like
-
     if combmode == 'paid':
-        LGR.info('Optimally combining data with parallel-acquired inhomogeneity '
-                 'desensitized (PAID) method')
-        combined = _combine_paid(data, tes)
+        LGR.info('Optimally combining data with parallel-acquired '
+                 'inhomogeneity desensitized (PAID) method')
     else:
         if t2s.ndim == 1:
             msg = 'Optimally combining data with voxel-wise T2* estimates'
         else:
             msg = ('Optimally combining data with voxel- and volume-wise T2* '
                    'estimates')
-        t2s = t2s[mask, ..., np.newaxis]  # mask out empty voxels/samples
-
         LGR.info(msg)
-        combined = _combine_t2s(data, tes, t2s)
+
+    mask = adaptive_mask >= 3
```

Comment:
For what I understand from this, there must be a minimum of 3 good echoes in order to keep a voxel. What happens with datasets that only have two echos?

Reply:
We generally don't support those, as the method of usage is quite different in e.g. the Bright and Murphy method (where the first echo is generally treated as a regressor). Maybe we should open a separate issue to confirm if that's mentioned in the documentation, and to highlight it as a condition of tedana ?